control/control.openSUSE.xml

```diff
@@ -143,36 +143,36 @@ textdomain="control"
 
             <!-- Replacement for EXTRAURLS and OPTIONALURLS -->
             <extra_url>
-                <baseurl>http://download.opensuse.org/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/tumbleweed/repo/oss/</baseurl>
                 <alias>repo-oss</alias>
-                <name>openSUSE-Factory-Oss</name>
+                <name>openSUSE-Tumbleweed-Oss</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">true</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/factory/repo/non-oss/</baseurl>
+                <baseurl>http://download.opensuse.org/tumbleweed/repo/non-oss/</baseurl>
                 <alias>repo-non-oss</alias>
-                <name>openSUSE-Factory-Non-Oss</name>
+                <name>openSUSE-Tumbleweed-Non-Oss</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">true</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/debug/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/debug/tumbleweed/repo/oss/</baseurl>
                 <alias>repo-debug</alias>
-                <name>openSUSE-Factory-Debug</name>
+                <name>openSUSE-Tumbleweed-Debug</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">false</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
```

Comment:
On Wed, 2015-01-28 at 08:58 -0800, Lukas Ocilka wrote:
> In control/control.openSUSE.xml:
> 
> >                  <prod_dir>/</prod_dir>
> >                  <enabled config:type="boolean">false</enabled>
> >                  <autorefresh config:type="boolean">true</autorefresh>
> >                  <priority config:type="integer">99</priority>
> >              </extra_url>
> >              <extra_url>
> > -                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
> > +                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
> 
> http://download.opensuse.org/source/tumbleweed/repo/oss/
> 
> Object not found!

This 404 has been fixed by now.
-- 
Dimstar / Dominique Leuenberger <dimstar@opensuse.org>